.github/workflows/build.yml

```diff
@@ -467,16 +467,21 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install pytest tqdm
+          python -m pip install pytest tqdm docopt commonmark
+          pip install --pre --find-links ./wheels/ "scikit-decide[all]"
```

Comment:
you are testing the last released skcikit-decide, not the one just being built.  Move your new test after pytest

Reply:
indeed...

Reply:
I will squash my corrections when you accept them.

Reply:
Please make changes as minimal as possible, there is no reason to move the "Install dependencies" stanza

Reply:
Ok. 
I just do not understand why scikit-decide is installed during "test" step rather than in "install" step. But indeed my changes are not minimal, I agree.
And i do understand why minimal changes are better.
I force pushed a single commit so that only minimal changes are displayed.

Reply:
>  And i do understand why minimal changes are better.

They are easier to review

Reply:
Yes exactly